src/Roslyn.Suppressions/Roslyn.Suppressions/RoslynSettings.cs

```diff
@@ -31,7 +30,7 @@ namespace SonarLint.VisualStudio.Roslyn.Suppressions
     /// </summary>
     public class RoslynSettings
     {
-        public static readonly RoslynSettings Empty = new RoslynSettings { Suppressions = Enumerable.Empty<SonarQubeIssue>() };
+        public static readonly RoslynSettings Empty = new RoslynSettings { Suppressions = Enumerable.Empty<SuppressedIssue>() };
```

Comment:
This is the root of the changes - switching to a new type that contains fewer fields.